constructor/construct.py

```diff
@@ -74,6 +74,12 @@
 List of package specifications, e.g. `python 2.7*`, `pyzmq` or `numpy >=1.8`.
 This list of specifications if given to the conda resolver (as if you were
 to create a new environment with those specs.
+'''),
+
+    ('user_requested_specs',                  False, (list, str), '''
+List of package specifications to be recorded as "user-requested" for the 
+initial environment in conda's history file. If not given, user-requested
+specs will fall back to 'specs'. 
 '''),
```

Comment:
Note the new config parameter.  We'll need to use this config parameter for Miniconda and Anaconda.  We should discuss what the values should be.

Reply:
Could we use this and drop the 'packages' option? It doesn't work anyway.